content/python/virtual_host.py.md

```diff
@@ -0,0 +1,423 @@
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### powerOffLiveGuest
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['SoftLayer_Virtual_Host']
+
+""""
+Virtual host id and uuid variable. 
+"""""
+id_host="1001547"
+uuid="12345678-bb9a-fe82-7239-4a33077cf32f"
+
+try:
+    response = virtual_service.powerOffLiveGuest(uuid,id=id_host)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### powerOnLiveGuest
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['SoftLayer_Virtual_Host']
+
+""""
+Virtual host id and uuid variable. 
+"""""
+id_host="1001547"
+uuid="12345678-bb9a-fe82-7239-4a33077cf32f"
+
+try:
+    response = virtual_service.powerOnLiveGuest(uuid,id=id_host)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### rebootSoftLiveGuest
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['SoftLayer_Virtual_Host']
+
+""""
+Virtual host id and uuid variable. 
+"""""
+id_host="1001547"
+uuid="12345678-bb9a-fe82-7239-4a33077cf32f"
+
+try:
+    response = virtual_service.rebootSoftLiveGuest(uuid,id=id_host)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### resumeLiveGuest
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+virtual_service = client['SoftLayer_Virtual_Host']
+
+""""
+Virtual host id and uuid variable. 
+"""""
+id_host="1001547"
+uuid="12345678-bb9a-fe82-7239-4a33077cf32f"
+
+try:
+    response = virtual_service.resumeLiveGuest(uuid,id=id_host)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+
+### Create virtual host
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+apiAuthenticationKey = client['SoftLayer_Product_Order']
+
+body_json = {
```

Comment:
Having this body_json object in a comment or something would be ok, but I'd prefer the examples show how to NOT use hard coded price-ids. I know its a little more work in the example, but I think it makes for a better overall example.

Ideally since this is a python example, you could just use the softlayer-python ordering manager to generate the order container. You can still pass that to placeOrder of course. I mostly just don't want hard coded price ids.

using either [Generate Order](https://softlayer-python.readthedocs.io/en/latest/api/managers/ordering/#SoftLayer.managers.ordering.OrderingManager.generate_order) or [Get Prices](https://softlayer-python.readthedocs.io/en/latest/api/managers/ordering/#SoftLayer.managers.ordering.OrderingManager.get_price_id_list) would be best I think.